CUEstion.WEB/ClientApp/src/app/services/questions.service.ts

```diff
@@ -160,4 +180,18 @@ export class QuestionsService {
       return this.http.put(this._serverAddress + `/questions/${questionId}/comments/${commentId}`, comment, options);
     }
   }
+
+  DeleteComment(questionId: number, answerId: number, commentId: number) {
+    const options = {
+      headers: new HttpHeaders({
+        Authorization: 'Bearer ' + localStorage.getItem('token')
+      })
+    };
+
+    if (answerId != null) {
+      return this.http.delete(this._serverAddress + `/questions/0/answers/${answerId}/comments/${commentId}`, options);
```

Comment:
probably I am missing something. can you explain me why "0" is in the route here?

Reply:
I've written 0 cause it doesn't matter, what number of question is, cause questions, answers, comments have their different unique Ids. In places, where I call such methods I do this due to lack of opportunity to give this methods necessary parameters, although they are optional